### DISCUSSION TITLE
Using Filebeat

### DISCUSSION THREAD

**Discussion by chongma** [docker-mailserver/docker-mailserver#2788] [Ideas]
# Subject

I would like to configure a not documented mail server use case

## Description

It seems that docker-mailserver had a filebeat configuration in the past.  Are there any instructions for monitoring the logs with filebeat now?  Is anyone still doing it?

**Comment by georglauterbach** [comment]
I am not sure about the configuration in the past, but currently, there is not Filebeat configuration and there are no plans to add it.

EDIT: I converted this into a discussion, maybe you have more luck with finding an answer here.

**Reply by chongma** [reply]
Thanks!  My use case is being able to spot anything unusual in the logs
